css/main_styles.css

```diff
@@ -0,0 +1,224 @@
+* {
+  list-style-type: none;
+  font-family: 'Lato', 'Merriweather', sans-serif;
+}
+
+h2 {
+  text-align: center;
+  margin-top: 3rem;
+  margin-bottom: 2rem;
+}
+
+#container {
```

Comment:
Great Job using CSS grid through out your whole document and having your code be DRY.